tests/integrations/sqlalchemy/test_sqlalchemy.py

```diff
@@ -118,7 +118,7 @@ class Address(Base):
     assert (
         render_span_tree(event)
         == """\
-- op=None: description=None
+- op=None: description='None'
```

Comment:
Why op is `None` and description is `'None'`? Other ops below have quotes.

Reply:
Hmm maybe `None` => JSON's `null`, while the description will always be a string?

Reply:
Ah yeah that's an unfortunate side-effect. Will switch to `json.dumps` as representation method